src/pynwb/core.py

```diff
@@ -1240,7 +1240,10 @@ def to_dataframe(self):
         data = {}
         for name in self.colnames:
             col = self.__df_cols[self.__colids[name]]
-            data[name] = col[:]
+            if isinstance(col.data, np.ndarray) and col.data.ndim > 1:
+                data[name] = [x for x in col[:]]
```

Comment:
I think this is the same as ``data[name] = col.tolist()`` correct? If so, I think using ``tolist`` is probably prefered

https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.ndarray.tolist.html

Reply:
`tolist` makes a list of lists.... I'm not sure if that's preferable to a list of numpy arrays. @bendichter ?

Reply:
Ok, I see. Thanks for the clarification, that makes sense.